File: Itido-BMS/dart/bms_dart/lib/src/models/address.dart
class Address {
  int id;
  String addressName;
  double lat;
  double lon;

  Address({
    this.addressName,
    this.lat,
    this.lon,
  });

  factory Address.fromJson(json) {
    if (json == null) return null;
    return Address._fromJson(json);
  }

  Address._fromJson(json)
      : this.id = json['id'],
        this.addressName = json['addressName'],
        this.lat = json['lat'],
        this.lon = json['lon'];

  Map<String, dynamic> toMap() => {
        'id': this.id,
        'addressName': this.addressName,
        'lat': this.lat,
        'lon': this.lon,
      };
}

File: Itido-BMS/RentAPI/Rent/Models/Projects/Address.cs
using System;
using System.Linq.Expressions;

namespace Rent.Models.Projects
{
    public class Address
    {
        public int ID { get; set; }
        public int ProjectItemID { get; set; }
        public virtual ProjectItem ProjectItem { get; set; }
        public string AddressName { get; set; }
        public double? Lat { get; set; }
        public double? Lon { get; set; }

        static public Expression<Func<Address, dynamic>> BasicDTO()
        {
            return v => v != null ?
            new
            {
                v.ID,
                v.AddressName,
                v.Lat,
                v.Lon,
            } : null;
        }
    }
}
